src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/metrics/booleanexpressioncomplexity/Example1.java

```diff
@@ -5,18 +5,20 @@
   </module>
 </module>
 */
+package com.puppycrawl.tools.checkstyle.checks.metrics.booleanexpressioncomplexity;
 
 // xdoc section -- start
-public class Test
+public class Example1
 {
   public static void main(String ... args)
   {
     boolean a = true;
     boolean b = false;
 
-    boolean c = (a & b) | (b ^ a);       // OK, 1(&) + 1(|) + 1(^) = 3 (max allowed 3)
+    boolean c = (a & b) | (b ^ a);       // OK, 1(&)+1(|)+1(^)=3 (max allowed 3)
 
-    boolean d = (a & b) ^ (a || b) | a;  // violation, 1(&) + 1(^) + 1(||) + 1(|) = 4
+    boolean d = (a & b) ^ (a || b) | a;  // violation,
+    // 1(&)+1(^)+1(||)+1(|)=4
```

Comment:
Can't we have this on the same line? If not, I would like something like:
```suggestion
    boolean d = (a & b) ^ (a || b) | a;  // violation,
                                         // 1(&)+1(^)+1(||)+1(|)=4
```

Reply:
That was initially the idea, yes !!
But, Checkstyle CI were throwing comments indentation errors. So, had to follow accordingly.

However, I beautified the file now. Let me know if you suggest otherwise